### PR TITLE
update index comments

### PR DESCRIPTION
**Pull request by msal4**
Comments do not reflect an up-to-date usage of indexes.
I'm new to ent. Let me know if i missed something.

### CODE REVIEW

File: schema/index/index.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func (b *Builder) StorageKey(key string) *Builder {
 //	func (T) Indexes() []ent.Index {
 //
 //		// Partial index on name where the entity is not deleted.
-//		index.Field("name").
+//		index.Fields("name").
 //			Annotations(index.Annotation{
 //				WhereClause: "deleted_at is not null",
```
Comment (msal4) [inline schema/index/index.go:110]: Should change these too but I'm not sure what to replace them with.

Comment (msal4) [reply schema/index/index.go:110]: maybe `field.Annotation{}`?

Comment (a8m) [reply schema/index/index.go:110]: I'll comment a suggestion.

File: schema/index/index.go [hunk 2]
```diff
@@ -105,7 +105,7 @@ func (b *Builder) StorageKey(key string) *Builder {
...
 //			})
```
Comment (a8m) [inline schema/index/index.go:111]:
```suggestion
//			Annotations(entsql.Prefix(100))
```

**Pull request merged**